index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@1&family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <p id="title">Mumenya Nyamu</p>
+        <ul class="link">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li>
+            <a href="#"><img src="assets/Icon-Mail.png" alt="" /></a>
+          </li>
+        </ul>
+      </nav>
+
+      <img id="header-image" src="assets/Illustration-header@2x.png" alt="" />
+      <div class="container">
+        <h1 class="item">Hey There. I’m Mumenya</h1>
+        <p class="item" id="support">I’am a Software Developer</p>
+        <p class="item" id="description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="social">
+        <ul class="link media">
+          <li>
+            <a href="#"
+              ><img id="github" src="assets/github@2x.png" alt=""
+            /></a>
+          </li>
+          <li>
+            <a href="#"><img src="assets/Linkedinicon@2x.png" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="assets/angellist@2x.png" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="assets/twitter@2x.png" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="assets/medium@2x.png" alt="" /></a>
+          </li>
+        </ul>
+      </div>
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly wrap all these contents in a `section class="headline"` tag as this is the first section in your portfolio to improve web accessibility.

Reply:
I have completed this. Your review would be highly appreciated.